browser/resources/brave_new_tab_page_refresh/components/top_sites/top_site_tile.tsx

```diff
@@ -16,154 +16,40 @@ function sanitizeTileURL(url: string) {
-        cancelAnimationFrame(dragInfo.current.updateFrame)
-      }
-      dragInfo.current.updateFrame = requestAnimationFrame(() => {
-        dragInfo.current.updateFrame = 0
-        const elem = rootRef.current
-        if (!elem) {
-          return
-        }
-        if (point) {
-          const rect = elem.getBoundingClientRect()
-          const location = getDropLocation(point.x, rect)
-          updateDropIndicator(elem, location)
-        } else {
-          updateDropIndicator(elem, null)
-        }
-      })
-    }, [])
-
   const { favicon, title, url } = props.topSite
 
   function onContextMenu(event: React.MouseEvent) {
-    event.preventDefault()
-  }
-
-  function onMouseUp(event: React.MouseEvent) {
-    if (event.button === 2) {
-      props.onRightClick(event)
-    }
-  }
-
-  function onDragStart(event: React.DragEvent<HTMLElement>) {
-    event.dataTransfer.setData('text/uri-list', url)
-    event.dataTransfer.setData('text/top-site-url', url)
-  }
-
-  function isTopSiteDrag(event: React.DragEvent) {
-    return event.dataTransfer.types.includes('text/top-site-url')
-  }
-
-  function onDragEnter(event: React.DragEvent<HTMLElement>) {
-    dragInfo.current.enterCount += 1;
-    if (dragInfo.current.enterCount === 1 && isTopSiteDrag(event)) {
+    if (props.onRightClick) {
       event.preventDefault()
     }
   }
 
-  function onDragOver(event: React.DragEvent<HTMLElement>) {
-    if (dragInfo.current.enterCount > 0 && isTopSiteDrag(event)) {
-      event.preventDefault()
-      event.dataTransfer.dropEffect = 'move'
-      if (rootRef.current) {
-        updateIndicator({ x: event.clientX, y: event.clientY })
-      }
-    }
-  }
-
-  function onDragLeave(event: React.DragEvent<HTMLElement>) {
-    dragInfo.current.enterCount -= 1;
-    if (dragInfo.current.enterCount <= 0) {
-      updateIndicator(null)
-    }
-  }
-
-  function onDrop(event: React.DragEvent) {
-    event.preventDefault()
-
-    dragInfo.current.enterCount = 0
-    updateIndicator(null)
-
-    const dragURL = event.dataTransfer.getData('text/top-site-url')
-    if (dragURL && rootRef.current) {
-      const rect = rootRef.current.getBoundingClientRect()
-      let location = getDropLocation(event.clientX, rect)
-      if (rootRef.current.matches(':dir(rtl)')) {
-        // Notify the parent of the drop's logical, rather than physical
-        // location. In RTL, this means we need to reverse the direction.
-        location = location === 'before' ? 'after' : 'before'
-      }
-      props.onDrop(dragURL, location)
+  function onMouseUp(event: React.MouseEvent) {
+    if (event.button === 2 && props.onRightClick) {
+      props.onRightClick(event)
     }
   }
 
   return (
     <a
-      ref={rootRef}
       className='top-site-tile'
       href={sanitizeTileURL(url)}
-      onDragEnter={onDragEnter}
-      onDragOver={onDragOver}
-      onDragLeave={onDragLeave}
-      onDrop={onDrop}
+      draggable={props.canDrag ? 'true' : 'false'}
```

Comment:
does this need to do the `true`/`false`? I think it'll be `.toString()`'d anyway

Reply:
Fixed - thanks!